src/write.rs

```diff
@@ -1002,12 +1002,28 @@ mod tests {
            ,-[<unknown>:1:1]
            |
          1 | apple ==
-           |         | 
-           |         `- Unexpected end of file
+           |          | 
+           |          `- Unexpected end of file
         ---'
         "###);
     }
 
+    #[test]
+    fn byte_spans_never_crash() {
+        let source = "apple\np\n\nempty\n";
+
+        for i in 0..source.len() {
+            for j in i..source.len() {
```

Comment:
The ranges should probably be `..=` here.

Reply:
Oh, good catch, is it possible to amend a PR?